examples/next/docs/react/demo/src/constants.ts

```diff
@@ -1,1205 +1,1108 @@
+  quantity: '30',
+  progress: 70,
+  rating: 3
+}, {
+  checked: false,
+  companyName: 'Minyx',
+  country: 'United States',
+  name: 'ML Road Tire',
+  sellDate: '07/07/2020',
+  orderId: '98-3938169',
+  inStock: true,
+  quantity: '73',
+  progress: 100,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Miboo',
+  country: 'China',
+  name: 'HL Road Tire',
+  sellDate: '25/07/2020',
+  orderId: '68-5197934',
+  inStock: true,
+  quantity: '158',
+  progress: 90,
+  rating: 1
+}, {
+  checked: false,
+  companyName: 'Realfire',
+  country: 'United States',
+  name: 'Touring Tire',
+  sellDate: '27/08/2020',
+  orderId: '39-8260460',
+  inStock: true,
+  quantity: '122',
+  progress: 50,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Shufflester',
+  country: 'China',
+  name: 'Mountain Tire Tube',
+  sellDate: '08/06/2020',
+  orderId: '45-9776170',
+  inStock: true,
+  quantity: '33',
+  progress: 20,
+  rating: 4
+}, {
+  checked: false,
+  companyName: 'Ntag',
+  country: 'China',
+  name: 'Road Tire Tube',
+  sellDate: '06/12/2020',
+  orderId: '45-0858451',
+  inStock: true,
+  quantity: '107',
+  progress: 60,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Jabberbean',
+  country: 'United States',
+  name: 'Touring Tire Tube',
+  sellDate: '26/04/2020',
+  orderId: '15-4247305',
+  inStock: true,
+  quantity: '15',
+  progress: 10,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Thoughtblab',
+  country: 'China',
+  name: 'LL Bottom Bracket',
+  sellDate: '21/05/2020',
+  orderId: '15-8534931',
+  inStock: true,
+  quantity: '168',
+  progress: 50,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Jabbertype',
+  country: 'China',
+  name: 'Classic Vest',
+  sellDate: '25/07/2020',
+  orderId: '23-1251557',
+  inStock: true,
+  quantity: '135',
+  progress: 40,
+  rating: 2
+}, {
+  checked: false,
+  companyName: 'Buzzshare',
+  country: 'United Kingdom',
+  name: 'Cycling Cap',
+  sellDate: '07/07/2020',
+  orderId: '86-5920601',
+  inStock: true,
+  quantity: '11',
+  progress: 10,
+  rating: 4
+}, {
+  checked: false,
+  companyName: 'Roodel',
+  country: 'United States',
+  name: 'Full-Finger Gloves',
+  sellDate: '13/01/2020',
+  orderId: '48-1055459',
+  inStock: true,
+  quantity: '41',
+  progress: 60,
+  rating: 4
+}, {
+  checked: false,
+  companyName: 'Zoovu',
+  country: 'China',
+  name: 'Half-Finger Gloves',
+  sellDate: '03/06/2020',
+  orderId: '12-7842022',
+  inStock: true,
+  quantity: '144',
+  progress: 60,
+  rating: 1
+}, {
+  checked: false,
+  companyName: 'Photofeed',
+  country: 'China',
+  name: 'HL Mountain Frame',
+  sellDate: '14/07/2020',
+  orderId: '94-5088099',
+  inStock: true,
+  quantity: '106',
+  progress: 10,
+  rating: 4
+}];
 
-export const SELECTED_CLASS = "selected";
-export const ODD_ROW_CLASS = "odd";
+export const SELECTED_CLASS = 'selected';
+export const ODD_ROW_CLASS = 'odd';
+
+export const MESSAGE = {
+  BAD_VALUE: '#bad-value#'
+};
```

Comment:
Do we need to introduce this variable using object?

```suggestion
export const BAD_VALUE_MESSAGE = '#bad-value#';
```